solidity/contracts/converter/LiquidityPoolV1Converter.sol

```diff
@@ -0,0 +1,488 @@
+pragma solidity 0.4.26;
+import './LiquidityPoolConverter.sol';
+import './interfaces/ITypedConverterFactory.sol';
+
+/*
+    LiquidityPoolV1Converter Factory
+*/
+contract LiquidityPoolV1ConverterFactory is ITypedConverterFactory {

```

Comment:
@yudilevi:
1. Why not in a separate file?
2. If not in a separate file, then why here and not in `ConverterFactory.sol`?
The incentives for #2 are:
1. Similar code (both contracts inherit from `ITypedConverterFactory`)
2. Short code (both contracts implement a single function `createConverter`)